data/pc/common/features.json

```diff
@@ -238,9 +238,17 @@
   },
   {
     "name": "hasEditBookPacket",
-    "description": "book editing is handled through a packet",
+    "description": "book editing is handled through a packet with the updated book item",
     "versions": [
       "1.13",
+	  "1.17.1"
+    ]
+  },
+  {
+    "name": "hasNonNbtEditBookPacket",
```

Comment:
The feature naming is confusing here. Can you elaborate what exactly the difference in packet is before vs after?

Reply:
The old edit book packet serializes the item and uses the nbt data from that
![image](https://github.com/PrismarineJS/minecraft-data/assets/16884313/0cdefc02-20ff-47b1-b73e-c271297ee969)

New packet sends the pages and title as packet fields
![image](https://github.com/PrismarineJS/minecraft-data/assets/16884313/f3eed84c-f22c-4a33-9938-4d6a9f779d08)

Reply:
How about adding a `editBookPacketUsesNbt` feature then? This seems clearer than the current naming between `hasEditBookPacket` and `hasNonNbtEditBookPacket`, as there is actaully an edit book packet on both version ranges.

Reply:
Done